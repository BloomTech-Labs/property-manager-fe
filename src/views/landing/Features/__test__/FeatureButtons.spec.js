import React from 'react';
import * as rtl from '@testing-library/react';
import '@testing-library/jest-dom';
import FeatureButton from '../FeatureButtons';

afterEach(rtl.cleanup);

test('Makes sure the word buy exists on buttons', async () => {
  const wrapper = rtl.render(<FeatureButton />);
  expect(await wrapper.queryAllByText(/buy/i));
});

test('snapshot', () => {
  const { container } = rtl.render(<FeatureButton />);
  expect(container.firstChild).toMatchSnapshot();
});
