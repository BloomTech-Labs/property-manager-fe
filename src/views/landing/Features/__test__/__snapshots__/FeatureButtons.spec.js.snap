// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  class="buy-btn-wrapper"
>
  <div
    class="empty"
  />
  <button
    class="buy-btn"
    type="submit"
  >
    Buy
  </button>
  <button
    class="buy-btn"
    type="submit"
  >
    Buy
  </button>
  <button
    class="buy-btn"
    type="submit"
  >
    Buy
  </button>
</div>
`;
